src/components/VolunteersForm.js

```diff
@@ -15,14 +15,22 @@ export default class VolunteersForm extends React.Component {
 
     constructor(props) {
       super(props);
-      this.state = { 
+      this.state = {
         minimumSelectionInterest: false,
-        minimumSelectionHAU: false
+        minimumSelectionHAU: false,
       };
     }
 
     handleOnChange = (event) => {
-      this.setState({ [event.target.name]: event.target.value });
+      if(trim(event.target.value) != ""){
+        document.getElementById(event.target.id).setCustomValidity("");
```

Comment:
Is this line necessary?

Reply:
Line 26 ? Yes, absolutely. Without it the form will not check again if the input is valid and always return an error tooltip

Reply:
Ah, gotcha!